tools/pot/openvino/tools/pot/graph/special_patterns.py

```diff
@@ -28,6 +28,21 @@ def create_swish_pattern():
     return pattern.set_name('swish_activation').pattern
 
 
+# Removing pattern se-blocks reduces accuracy
```

Comment:
Can you please change it to docstring style and place of each pattern-builder function?
Like here,
```
def create_se_pattern()
    """Removing this pattern can drop accuracy after quantization of model w/ SE-blocks"""
    ...
```

The same for the second function.